scripts/pack.mjs

```diff
@@ -82,9 +93,11 @@ async function readFiles(p) {
     }, {});
 }
 
-console.log('copy local assets');
-copyFolderRecursiveSync('./assets', './dist/')
-copyFolderRecursiveSync('./css', './dist/')
+if (packAssets) {
+    console.log('copy local assets');
+    copyFolderRecursiveSync('./assets', './dist/')
+    copyFolderRecursiveSync('./css', './dist/')
+}
```

Comment:
I would always do this copy.

It will copy the css and manifest files, those files aren't big and if you change something in the manifest or the styling you would need to run the other command that's extracting everything.

Reply:
Yeah, the css sheets and manifest should be copied every time because they're "code".
But re-copying all the jpgs and pngs takes time and is completely unnecessary.

Reply:
the images are not in the `./assets` folder. they get extracted from temp directly to `./dist/assets/` Athleast they should...

Reply:
There are image in my assets folder...